specs/StatusBarActivity.md

```diff
@@ -0,0 +1,155 @@
+<!-- USAGE
+  * Fill in each of the sections (like Background) below
+  * Wrap code with `single line of code` or ```code block```
+  * Before submitting, delete all <!-- TEMPLATE marked comments in this file,
+    and the following quote banner:
+-->
+# Background
+The browser has a Status bar that displays text when hovering over a link, or performing some activity. Currently, 
+developers are able to opt in to disable showing the status bar through the browser 
+settings.
+
+Developers would also like to be able to opt in to intercept the updates which would 
+normally be displayed by the Status bar, and show it using thier own custom UI. 
+# Description
+We propose two new events for WebView2 that would allow developers to 
+listen for Status bar updates which are triggered by activity on the WebView, and then handle those updates however they want in their applications.
+
+Developers will be able to: 
+
+1) Register an even handler for knowing when to show the status bar and what text to show
+2) Register an handler for knowing when the browser wants to dismiss the status bar
+
+
+# Examples
+## Win32 C++ Registering a listener for status bar showing
+```
+CHECK_FAILURE(m_webView->add_StatusBarShowing(
```

Comment:
Are you also updating the sample code in our sample apps? If so, you can use the code directly from that for the samples here